test/functionalTest/cases/0000_deprecated_checkings/log_deprecate_warning.test

```diff
@@ -50,9 +50,9 @@ echo
 echo
 
 
-echo "02. GET /v1/contextEntities/E"
-echo "============================="
-orionCurl --url /v1/contextEntities/E
+echo "02. GET /v1/contextEntities/E/attributes/A"
+echo "=========================================="
+orionCurl --url /v1/contextEntities/E/attributes/A
```

Comment:
The NGSIv1 operation without payload we used in this test was deprecated, so we have changed to another one.

NTC (informative)